www.foia.gov/_sass/modules/_sidebar_timeline.scss

```diff
@@ -0,0 +1,31 @@
+.sidebar_progress-bar {
+  margin: 0 0 0 0.8rem; // to accommodate left half of bubbles
+  border-left: 1px solid $color-primary-alt; // line between bubbles
+  li {
+    line-height: 0.95; // to get the bubble/text to line up
+    margin-left: $space-1x * -1; // to line the bubbles halfway on the ul's border
+    margin-bottom: $space-3x;
+  }
+  a {
+    &:before {
+      width: $space-2x;
+      height: $space-2x;
+      display: block;
+      float: left;
+      content: " ";
+      background-color: $color-primary-darkest;
+      border: 1px solid $color-primary-alt;
+      border-radius: 100px;
+      margin-right: $space-2x;
+    }
+    &.step_active {
+      font-weight: bold;
+      &:before {
+        background-color: $color-primary-alt;
+      }
+    }
+    span {
+      display: block;
+    }
```

Comment:
Why not just have these as divs?

Reply:
The `line-height` seems to behave better this way, for reasons that I don't completely understand. (Block-level and inline elements treat `line-height` differently, but you'd think setting `display: block` on an inline element would make it render like a block-level element? It doesn't seem to, and I'm not sure why. CSS is cray.)